sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -136,26 +184,26 @@
 <SALTelemetry>
     <Subsystem>MTM2</Subsystem>
     <EFDB_Topic>MTM2_temperaturesMeasured</EFDB_Topic>
-    <Description>Mirror and cell temperature sensor readings.</Description>
+    <Description>Mirror and cell temperature sensor readings relative to the origin (offset).</Description>
     <item>
         <EFDB_Name>ringTemperature</EFDB_Name>
```

Comment:
I just noticed that the attributes also have "temperature" on them.. I would recommend removing "Temperature" from the topic attributes because we already know they are temperatures from the topic name. 

So, instead of `ringTemperature` just `ring` would be fine, and so on...